bears/apertium/ApertiumLintBear.py

```diff
@@ -0,0 +1,361 @@
+import json
+
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.PipRequirement import PipRequirement
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+
+
+@linter(executable='apertium_lint',
+        output_format='regex',
+        output_regex=r'(?P<severity>\w+) : (?P<message>[^:\n]+: '
+                     r'(?P<origin>[^\n]+))\n[^:\n]+: (?P<line>\d+)',
+        config_suffix='.json',
+        severity_map={'Warning': RESULT_SEVERITY.MAJOR})
+class ApertiumLintBear:
+    """
+    `Apertium lint` is a python module that lints out irregular yet
+    acceptable constructs that may creep into files involved in the
+    transfer process. The lint is designed specifically to handle
+    4 file types dictionaries, transfer, modes and tagger.
+
+    Lints the file using
+    `apertium_lint <https://svn.code.sf.net/p/apertium/svn
+    /trunk/apertium-tools/apertium-lint/>`.
```

Comment:
Maybe better link to https://pypi.python.org/pypi/apertium_lint instead. It might be more appealing and also links to the above URL

And also add `SEE_MORE` according to https://github.com/coala/coala-bears/issues/1136